.env.example

```diff
@@ -42,6 +42,15 @@ MAIL_FROM_NAME=
 MAILGUN_DOMAIN=
 MAILGUN_SECRET=
 
+# Slack Webhook Endpoint path to be appended to https://hooks.slack.com/services/.
+SLACK_ENDPOINT=
+
+# Default slack channel for the bot to post in.
+SLACK_CHANNEL=
+
+# Default username to post as
+SLACK_USERNAME=
```

Comment:
I would dispute the "no need for comments" in this file entirely. I feel at the very least each configuration block should have a block comment explaining what it's for. At present, it's dependent on the developer reading the CONTRIBUTING.md or actual config files themselves which seems counter productive. It makes more sense, to me, to have comments embedded next to where you set values as opposed to relying on some second file to get the details/consequences of setting a config var. E.g. 
```
# The VATSIM_CERT options are used to connect to the single sign-on system. Each value is 
# unique to a division, and a secret so leave them blank
SSO_CERT=
SSO_KEY=VATSIM_UK
SSO_SECRET=
```
It may seem superfluous but it removes doubt and ambiguity right where they're set. 

Regarding the use of "BOT" my logic was that the purpose of the slack configuration is to hook into slack. Core uses it for the purposes of a BOT (which could be highlighted in the comments) but by defining the vars as "BOT" you're constraining the scope of the environment variable. Again, some may see it as superfluous but in my head there's a clear separation.